.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+on:
+  push:
+    branches:
+      - '*'
+      - '!dev'
+      - '!master' # remove once main branch is dev
+jobs:
+  build-and-test:
+    name: Run Build and UI Tests
+    runs-on: macos-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Build
+        working-directory: ./BrickKit
+        run: ./gradlew build
+      
+      - name: Run Tests
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 29  
```

Comment:
should this run on minsdk as well?

Reply:
Trie that but it seems to blow up inside mockito mockmaker. Created https://github.com/wayfair/brickkit-android/issues/147 to add that